src/main/resources/data/retail/transactions-currentaccount.json

```diff
@@ -11,13 +11,13 @@
     "reference": "czyo9c2n7z",
     "type": "Withdrawal",
     "typeGroup": "Payment",
-    "description": "Fashion shopping",
+    "description": "Christmas shopping",
```

Comment:
Description should be `Deposit to <Pocket name>`, see this ticket for details: https://backbase.atlassian.net/browse/MAINT-13092

Reply:
Thanks for this info. Changed descriptions.